python/python-core/src/main/python/domain/_annotations.py

```diff
@@ -231,7 +231,7 @@ def __init__(self, *,
         super().__init__(JavaPiggybackShadowVariable,
                          {
                              'shadowVariableName': PythonClassTranslator.getJavaFieldName(shadow_variable_name),
-                             'shadowEntityClass': shadow_entity_class,
+                             'shadowEntityClass': get_asm_type(shadow_entity_class),
```

Comment:
`get_asm_type` is not needed; remove.

Reply:
Removing `get_asm_type` will cause the test to fail. I'm concerned that other parts of the code, that use the source class, may also fail, and the existing tests do not cover it.

```
E       java.lang.java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: The entityClass (class org.jpyinterpreter.user.tests.test_vehicle_routing.Visit) has a @PiggybackShadowVariable annotated property (piggyback_arrival_time) with a shadowEntityClass (interface ai.timefold.jpyinterpreter.PythonLikeObject) which is not a valid planning entity.
E       Maybe check the annotations of the class (interface ai.timefold.jpyinterpreter.PythonLikeObject).
E       Maybe add the class (interface ai.timefold.jpyinterpreter.PythonLikeObject) among planning entities in the solver configuration.

.tox/p312/lib/python3.12/site-packages/timefold/solver/_solver_factory.py:52: java.lang.IllegalArgumentException

```